utils/mk-consul-env

```diff
@@ -81,8 +81,12 @@ sed -r -e "s/^(NODE=).*/\1$(node-name)/" \
     sudo tee $ENV_FILE >/dev/null
 
 if [[ -n $join_addr ]]; then
-    sudo sed -r -e "s/^(JOIN=).*/\1${join_addr:+-retry-join $join_addr}/" \
-             -i $ENV_FILE
+    # join_addr is not empty and it contains one or more ip addresses separated by space.
+    # Multiple server addresses must turn into multiple '-retry-join ipN'.
+    # For instance '127.0.0.1 192.168.0.1' => '-retry-join 127.0.0.1 -retry-join 192.168.0.1'
+
+    retry_str="$(echo "$join_addr" | tr ' ' '\n' | sed 's/\(^.*$\)/-retry-join \1/' | tr '\n' ' ')"
```

Comment:
This might be simpler (without the need for `tr`):
```
$ echo aa b c | sed 's/\(\<[[:alnum:]]\)/-retry-join \1/g'
-retry-join aa -retry-join b -retry-join c
```

Reply:
Your version looks too cryptic. IMO the version with `tr` is longer but easier to understand.

@mssawant, @suvratjoshi What's your opinion?

Reply:
I am okay with `tr` too, patch as it is did not work as expected for me. Please find my comments and suggested changes.

Reply:
Nevermind, `[:alnum:]` didn't work for IP addresses anyway. I've re-written this with `awk` to address @mssawant's finding.